src/pages/Login/SignMode/Input/Input.js

```diff
@@ -0,0 +1,23 @@
+import './Input.scss';
+const Input = ({ sign, inputValue, saveInput }) => {
+  return (
+    <>
+      {sign.map((input, idx) => {
+        const type = input.text.includes('패스워드') ? 'password' : 'text';
+        return (
```

Comment:
```suggestion
  return (
    <>
      {sign.map(({text, name}, idx) => {
        const type = input.text.includes('패스워드') ? 'password' : 'text';
        return (
```
이렇게  map 메서드 내부에서도 구조분해할당을 할 수 있습니다.

Reply:
오,, 여기까지 생각도 못했습니다.!! 리뷰반영했습니다.